README.md

```diff
@@ -1,2 +1,61 @@
-# Project
-DAY2 BLOCK WEEK1
+# Hello Microverse
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+
+
+One page website
+
+
+## Built With
+
+- html 
+- css
+- visual studio.NET
+- github
+
+
+
+
+## Getting Started
+
+**This is an example of how you may give instructions on setting up your project locally.**
+**Modify this file to match your project, remove sections that don't apply. For example: delete the testing section if the currect project doesn't require testing.**
+
```

Comment:
- Kindly remove this text block as these are instructions meant for you to follow to add instructions on how the users could have your project on their local machine.  Update to this: 
`To get a local copy up and running follow these simple example steps.`
Then you can now add your steps..